provisioning/transport/amqp/test/_amqp_test.js

```diff
@@ -87,6 +87,32 @@ describe('Amqp', function () {
     amqp = null;
   });
 
+  describe('Symmetric Key', function() {
+    /*Tests_SRS_NODE_PROVISIONING_AMQP_06_001: [ The sas token passed shall be saved into the current transport object. ] */
+    it('setSasToken', function(callback) {
+      var fakeSas = 'fake sas';
+      amqp.setSasToken(fakeSas);
+      assert(amqp._sasToken, fakeSas);
+      callback();
+    });
```

Comment:
testing internal implementation details - better test would be to call setSasToken, then trigger an operation that uses it and verify that it's is actually being used.

Reply:
Which oddly enough is what the next unit test does.  I'll delete this one then.

---
In reply to: [231363962](https://github.com/Azure/azure-iot-sdk-node/pull/418#discussion_r231363962) [](ancestors = 231363962)